advanced-examples/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "advanced-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx @elastic/synthetics journeys",
+    "push": "npx @elastic/synthetics push --auth $SYNTHETICS_API_KEY"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@elastic/synthetics": "^1.0.0-beta.31"
+  },
+  "devDependencies": {
+    "@types/node": "^18.7.11"
```

Comment:
Did the init command added this types by default and also the lock files?

Reply:
No, I added it. We should add that to the init command, otherwise vs code's completion was broken

Reply:
To be more specific it's required by our use of `Buffer`